api/src/main/java/org/apache/cloudstack/api/command/user/snapshot/CopySnapshotCmd.java

```diff
@@ -84,6 +85,15 @@ public class CopySnapshotCmd extends BaseAsyncCmd implements UserCmd {
                     "Do not specify destzoneid and destzoneids together, however one of them is required.")
     protected List<Long> destZoneIds;
 
+    @Parameter(name = ApiConstants.STORAGE_ID_LIST,
+            type=CommandType.LIST,
+            collectionType = CommandType.UUID,
+            entityType = StoragePoolResponse.class,
+            required = false,
+            description = "A comma-separated list of IDs of the storage pools in other zones in which the snapshot will be made available. " +
+                    "The snapshot will always be made available in the zone in which the volume is present.")
```

Comment:
```suggestion
                    "The snapshot will always be made available in the zone in which the volume is present. Currently supported for StorPool only")
```

update wherever applicable if ok

Reply:
added in the last commit